README.md

```diff
@@ -1,26 +1,30 @@
 # Flashbots
 
-Flashbots is a research and development organization working on mitigating the negative externalities of Maximal Extractable Value (MEV) and avoiding the existential risks MEV could cause to stateful blockchains like Ethereum. Our primary focus is to enable a *permissionless*, *transparent*, and *sustainable* ecosystem for MEV, via a three-pronged approach: 
+Flashbots is a research and development organization formed to mitigate the negative externalities posed by Maximal Extractable Value (MEV) to stateful blockchains, starting with Ethereum. 
+
+## Our Work
+Our primary focus is to enable a *permissionless*, *transparent*, and *sustainable* ecosystem for MEV, via a three-pronged approach: 
 
 * **Illuminate**: Bringing transparency to MEV activity.
 * **Democratize**: Democratizing access to MEV revenue.
 * **Distribute**: Enabling sustainable distribution of MEV revenue.
 
-Our efforts are separated into three sub-organizations:
-* **Product**: builds core infrastructure and ecosystem tooling that redefines the block production supply chain. This includes [MEV-Boost](https://boost.flashbots.net/), [SUAVE](https://writings.flashbots.net/the-future-of-mev-is-suave) as well as [Flashbots Protect RPC](https://docs.flashbots.net/flashbots-protect/overview), [mev-geth](https://github.com/flashbots/mev-geth), [MEV-Inspect](https://docs.flashbots.net/flashbots-data/mev-inspect-py/overview) and [MEV-Explore](https://explore.flashbots.net).
-* **Research**: an open, transparent, and [collaborative research effort](https://github.com/flashbots/mev-research) that explores MEV market dynamics and blockchain design challenges at the intersection of economics, security, and cryptography. To fuel the collaborative nature of our research, anybody can submit a [Flashbots Research Proposal](https://github.com/flashbots/mev-research/blob/main/process.md) (FRPs) and might become eligible for a grant.
-* **Coordination**: an organizational effort to streamline operational processes, interface with external teams in the ecosystem and scale ourselves to maximize our impact.
+
+## Our Dual Engines
+Product & research are the semi-autonomous dual engines that propel our organization forward:
+- [Flashbots Product](https://docs.flashbots.net/) builds core infrastructure and ecosystem tooling that redefine the block production supply chain across blockchains.
+- [Flashbots Research](https://github.com/flashbots/mev-research) explores MEV market dynamics and blockchain design challenges at the intersection of economics, security, and cryptography.
 
 Flashbots Research and Product are tightly-coupled dual engines that propel our organization forward in a phased approach:
```

Comment:
Needlessly repetitious imo. One sentence about dual engines is enough :sweat_smile: